client/app/components/packages/applicant-fieldset.js

```diff
@@ -1,7 +1,32 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
+import { STATE_OPTION_SET } from '../../models/applicant';
 
 export default class ApplicantFieldset extends Component {
+  get stateOptions() {
+    return Object.values(STATE_OPTION_SET);
+  }
+
+  // TODO: Use a global helper instead after writing
+  // unified optionset handling
+  get stateLabelLookup() {
+    const lookup = {};
+    const { stateOptions } = this;
+
+    for (let i = 0; i < stateOptions.length; i += 1) {
+      lookup[stateOptions[i].code] = stateOptions[i];
+    }
+
+    return lookup;
+  }
```

Comment:
You might add a comment "key the lookup by its code name"

Are there any other ways to do this? 

Alternative to consider: 
```javascript
  get stateLabelLookup() {
    return this.stateOptions
      .reduce((accumulator, option) => ({
        [option.code]: option,
        ...accumulator,
    }), { });
  }
```

```javascript
  get stateLabelLookup() {
    return this.stateOptions
      .reduce((accumulator, option) => {

        return {
          [option.code]: option,
          ...accumulator,
        }
      }, {});
  }
```

Reply:
Made some changes!